public/js/circos.js

```diff
@@ -429,7 +429,7 @@ class Graph {
         }, this));
         _.each(this.chords_arr, function (obj) {
             $('#' + obj[0] + '_' + obj[3]).attr('data-toggle', 'tooltip')
-                .attr('title', 'Identity: ' + obj[7].identity + '<br> E value: ' + Helpers.prettify_evalue(obj[7].evalue));
+                .attr('title', 'Identity: ' + obj[7].identity + '%' + '<br> E value: ' + Helpers.prettify_evalue(obj[7].evalue));
```

Comment:
This is incorrect. The value of identity provided by BLAST is a count of the number of matching residues. You cannot just add a % to the end of a count.

Reply:
(hsp.identity / hsp.length) * 100 

rounded to two decimal places